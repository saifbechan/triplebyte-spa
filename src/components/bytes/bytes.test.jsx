import React from 'react';
import { Provider } from 'react-redux';
import { render, cleanup } from '@testing-library/react';
import configureStore from 'redux-mock-store';

import Bytes from './bytes.component';

const mockStore = configureStore([]);

let store;

beforeEach(() => {
  store = mockStore({ bytes: { values: [0, 0, 0, 1, 0, 0, 1, 1, 0, 1] } });
});

afterEach(cleanup);

it('renders correctly', () => {
  const { asFragment } = render(
    <Provider store={store}>
      <Bytes />
    </Provider>
  );
  expect(asFragment()).toMatchSnapshot();
});
