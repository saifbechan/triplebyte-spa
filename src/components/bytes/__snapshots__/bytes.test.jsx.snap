// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-AykKD emnyUS"
  >
    <div
      class="sc-AykKC hZQdx"
    >
      0
    </div>
    <div
      class="sc-AykKC hZQdx"
    >
      0
    </div>
    <div
      class="sc-AykKC hZQdx"
    >
      0
    </div>
    <div
      class="sc-AykKC hZQdx"
    >
      1
    </div>
    <div
      class="sc-AykKC hZQdx"
    >
      0
    </div>
    <div
      class="sc-AykKC hZQdx"
    >
      0
    </div>
    <div
      class="sc-AykKC hZQdx"
    >
      1
    </div>
    <div
      class="sc-AykKC hZQdx"
    >
      1
    </div>
    <div
      class="sc-AykKC hZQdx"
    >
      0
    </div>
    <div
      class="sc-AykKC hZQdx"
    >
      1
    </div>
  </div>
</DocumentFragment>
`;
